src/sERC20.sol

```diff
@@ -8,6 +8,7 @@ import { ERC20 } from "openzeppelin-contracts/contracts/token/ERC20/ERC20.sol";
 /// @dev Synthetic ERC20 tokens out of 1155a
 contract sERC20 is ERC20 {
     address public immutable ERC1155A;
+    uint8 private immutable tokenDecimals;
```

Comment:
there is no interface file for sERC20 e.g. ISERC20.SOL, might be useful to have

Reply:
added